spec/models/package_category_spec.rb

```diff
@@ -1,4 +1,15 @@
 require 'rails_helper'
 
 RSpec.describe PackageCategory, type: :model do
+  describe 'Association' do
+    it { is_expected.to have_many :package_sub_categories }
```

Comment:
@swatijadhav cc @mcm-ham am I right in thinking ```package_sub_categories``` is a join table? If so, should it be called ```package_categories_package_types``` ?